lib/awful/client.lua

```diff
@@ -900,11 +900,11 @@ function client.setwfact(wfact, c)
     t:emit_signal("property::windowfact")
 end
 
---- Change a client's window factor.
+--- Change window factor of a client.
 --
 -- @legacylayout awful.client.incwfact
 -- @tparam number add Amount to increase/decrease the client's window.
---   Should be between 0.0 and 1.0.
+--   Should be between -1.0 and 1.0 (all window factors get normalized to 1).
```

Comment:
Used your phrasing now.  Leaving it out is not really an option: the documentation should explain what it is about after all - which it is still not great at, but better than before.